gtsfm/view_graph_estimator/view_graph_estimator_base.py

```diff
@@ -319,9 +322,13 @@ def create_computation_graph(
             view_graph_edges=view_graph_edges,
         )
 
+        pruned_i2Ri1_graph, pruned_i2Ui1_graph = dask.delayed(graph_utils.prune_to_largest_connected_component, nout=2)(
```

Comment:
@ayushbaid @johnwlambert I don't think this is a change of behavior as it is how its currently done. But, is it okay to pass downstream correspondences/two-view reports for which cameras are not in the pruned graph?